components/object-property-list/object-property-list-item.js

```diff
@@ -67,6 +78,10 @@ export class ObjectPropertyListItem extends RtlMixin(LitElement) {
 			</span>
 		`;
 	}
+
+	_renderText() {
+		return html`<span class="d2l-skeletize" aria-hidden=${this.skeleton ? 'true' : 'false'}>${this.text}</span>`;
```

Comment:
Nit:
```suggestion
		return html`<span class="d2l-skeletize" aria-hidden="${this.skeleton ? 'true' : 'false'}">${this.text}</span>`;
```

Reply:
out of curiosity, is this a style preference, or is there a functional difference?

Reply:
Like technically all attributes in HTML are strings, so should be quoted. Lit may be doing some magic here to fix things for you, but if you missed quotes and the value resolved to something with a space in it, then you could end up with: 

```html
<d2l-object-property-list-item text=hello there what's up skeleton>
```

In that case, the browser would interpret the value as "hello" and "there", "what's", "up" and "skeleton" as boolean attributes, I think? Lit may be fixing things for you though like I said, but IMO it's cleaner and safer to just use the quotes.

Reply:
Unquoted is valid HTML, in HTML5+:
https://html.spec.whatwg.org/multipage/syntax.html#attributes-2

As mentioned [here](https://stackoverflow.com/a/13056802), there are cases where it was required pre-HTML5. But yes we still want to use them for consistency and to avoid issues with whitespace.

Reply:
Haha so I guess "technically" was the wrong word to use, thanks for linking to that. So _technically_ they can be unquoted if we're certain that the values don't contain whitespace. In practice, it's easier IMO to just always quote them.